docs/support.md

```diff
@@ -0,0 +1,12 @@
+## Currently Windows and WSL2 do not support `yast`.
+
+## Issue with Windows support
```

Comment:
1.FileSystem -> File System
2.the config files of yast had no support in the Windows File System. -> the config utils will not exibit same behaviour of Windows File System

Reply:
I think it looks fine now, please check.